docs/development/logging.md

```diff
@@ -133,7 +133,7 @@ results in
 2021-12-16T09:35:59.099+0100    INFO    controller.shoot    Reconciling Shoot        {"name": "sunflower", "namespace": "garden-greenhouse"}
 ```
 
-The logger is injected by controller-runtime's `Controller` implementation and our `controllerutils.CreateWorker` alike (if a logger is passed using `controllerutils.WithLogger`). The logger returned by `logf.FromContext` is never `nil`. If the context doesn't carry a logger, it falls back to the global logger (`logf.Log`), which might discard logs if not configured, but is also never `nil`.
```

Comment:
There are more occurrences of `CreateWorker` in this file.

https://github.com/gardener/gardener/blob/bb4f4885918a7366930abe8dc2c01970b898ce5c/docs/development/logging.md#L140
https://github.com/gardener/gardener/blob/bb4f4885918a7366930abe8dc2c01970b898ce5c/docs/development/logging.md#L240

Can we adapt them as well?